Lesson9/anna/styles.css

```diff
@@ -0,0 +1,279 @@
+    align-items:center;
+    margin:var(--space-sm);
+    gap:var(--space-xs);
+  }
+  
+  .first{
+    display:flex;
+    align-items:center;
+  }
+  .second{
+    display:flex;
+    align-items:center;
+  }
+    
+  
+  
+  .article-actions-right{
+    display:flex;
+    align-items:center;
+    gap:var(--space-md);
+     margin:var(--space-sm);
+  }
+  
+  .hero-content{
+    max-width:100%;
+    height: auto;
+    display: block;
+    margin: 0 auto;
+  }
+  .fig-text{
+    color:var(--color-secondary);
+    font-family:sohne, "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: var(--font-size-m);
+    margin:var(--space-xs) auto;
+    text-align:center;
+    display:block;
+    font-weight:400;
+  }
+  .fig-text-link{
+    text-decoration:underline;
+    color:var(--color-secondary);
+  }
+  
+  .parag{
+    max-width:680px;
+    color:inherit;
+    font-size:var(--font-size-xl);
+    font-family: source-serif-pro, Georgia, Cambria, "Times New Roman", Times, serif;
+    line-height:32px;
+    letter-spacing: -0.003em;
+    padding:20px;
+    margin:0 auto;
+  }
+  .scnd{
+    margin-top:42px;
+    margin-bottom:10px;
+  }
+  
+  
+  .space{
+    margin-top:46px;
+    margin-bottom:6px;
+  }
+  
+  .fth{
+    margin-top:18px;
+    margin-bottom:10px;
+  }
+  .oth-photo{
+    max-width:100%;
+    height: auto;
+    display: block;
+    margin: 0 auto;
+  } /*bunu yaparak fotoğrafı büyük yapmaya çalıştım ama olmadı*/
+  
+  .codeblc{
+    display:block;
+    background-color:var(--codebackground);
+    font-family:source-code-pro, Menlo, Monaco, "Courier New", Courier, monospace;
+    padding:var(--space-sm);
+    font-size:var( --font-size-l);
+    margin-top:50px;
+    line-height: 1.18;
+  }
+  .adv:hover{
+    background:#00FF00;
+  }
+  .litblock{
+    background-color:var(--codebackground);
+  }
+  .line{
+    text-decoration:underline;
+    color:inherit;
+  }
+  .caption{
+    color:var(--color-secondary);
+    font-size:var(--font-size-m);
+  }
+  .cen{
+    width:100%;
+      height:auto;
+      margin:0px;
+      padding:0px;
+      display:block;
+  }/*bunu yaparak fotoğrafı büyük yapmaya çalıştım ama olmadı*/
+  .num{
+    font-size:20px;
+  }
+  .box{
+    background-color:var(--codebackground);
+    border-radius:100px;
+    display:inline-block;
+    padding:8px 16px;
+    border:1px solid var(--codebackground);
+    font-size:14px;
+    margin:24px 0px; 
+  }
+  .last-container{
+    display:flex;
+    justify-content:space-between;
+  }
+  
+  @media all and (max-width: 767.99px) {
+    .width-restricted {
+      padding: 0 24px;
+    }
+    .article-author-info-container {
+      gap: var(--space-xs);
+    } 
+    .hero-content{
+      width:100%;
+      height:auto;
+      margin:0px;
+      padding:0px;
+      display:block;
+    }
+  
+    .oth-photo{
+      width:100%;
+      height:auto;
+      margin:0px;
+      padding:0px;
+      display:block;/*bunu yaparak fotoğrafı büyük yapmaya çalıştım ama olmadı*/
```

Comment:
This picture's original size is 560px so it's better never to stretch the image larger than it's original size. You would normally need to use image is larger size (e.g. up to 1000px wide) for the desktop version of this page and a smaller version for smaller device sizes. 
Here, you can remove everything from width and height properties. 
Ideally, it would be like this instead:
```
    .oth-photo{
      max-width: 100%;
      height:auto;
    }
```